wdl-doc/src/parameter.rs

```diff
@@ -41,22 +109,73 @@ impl Parameter {
         self.decl.name().text().to_owned()
     }
 
-    /// Get the type of the parameter.
-    pub fn ty(&self) -> String {
-        self.decl.ty().to_string()
+    /// Get the meta of the parameter.
+    pub fn meta(&self) -> &MetaMap {
+        &self.meta
     }
 
-    /// Get whether the parameter is an input or output.
-    pub fn io(&self) -> InputOutput {
-        self.io
+    /// Get the type of the parameter as a String.
```

Comment:
```suggestion
    /// Get the type of the parameter as a string.
```

Either say `[``String``]` or just `string` here.